src/librustc_trans/back/link.rs

```diff
@@ -712,6 +713,11 @@ fn link_natively(sess: &Session,
         cmd.arg(root.join(obj));
     }
 
+    if sess.target.target.options.is_like_emscripten &&
+       sess.panic_strategy() == PanicStrategy::Abort {
+        cmd.arg("-s DISABLE_EXCEPTION_CATCHING=1");
```

Comment:
I think this'll want to get passed as two separate arguments, right? (right now this is just being passed as one)